src/ui/common/src/reducers/workflow.ts

```diff
@@ -46,6 +48,7 @@ export type OperatorResult = {
 
 export type WorkflowState = {
   loadingStatus: LoadingStatus;
+  savedObjects: { [id: string]: SavedObject[] };
```

Comment:
since the request-response is handled separately than workflow load, let's add a separate `loadingStatus` to handle the status of these objects